packages/core/src/Background.ts

```diff
@@ -0,0 +1,32 @@
+import { Color } from "@oasis-engine/math";
+import { Sky } from "./sky/Sky";
+
+/**
+ * The Background mode enumeration.
+ */
+export enum BackgroundMode {
```

Comment:
Use seperate file , for example `*/enums/BackgroundMode.ts` path.

Reply:
+1